### DISCUSSION TITLE
Asustor Release Timeline

### DISCUSSION THREAD

**Discussion by krakky4** [photoprism/photoprism#4086] [Q&A]
Hello,
thanks for a powerful and well designed photo management app.
Is there a schedule for releases on the Asustor App-Store? The latest version I can find there is from October and there seem to have been at least 2 updated versions since then.
Thanks and bye

**Comment by lastzero** [comment]
You're absolutely right! I'll get in touch with them to let them know there's a newer release.

**Reply by master3395** [reply]
> You're absolutely right! I'll get in touch with them to let them know there's a newer release.

Couldn't we also use the releases and manually install the app, if the APK was available on the github release also?

**Reply by graciousgrey** [reply]
The PhotoPrism app in the Asustor app store is maintained by the Asustor team.

**Reply by master3395** [reply]
> The PhotoPrism app in the Asustor app store is maintained by the Asustor team.

Do you know if it is possible to use a folder that doesn't have the app as the owner? (The Asustor Image App (photogallery 3), uses the user's own directory to store the images, so other users don't have access to them.
I did try to link the directory from the docker folders import folder, but it didn't seem like it was working.

Because Asustor is quite stubborn only the user is the owner and not the admin, and the group is users, not administrators.
I did try and set the photo app's folder permissions to admin and administrators but was refused because once i opened the app on the phone, the NAS over-wrote the permissions.

Owner: admin [999]
Group: administrators [999]
/volume1/Docker/PhotoPrism/data/import/
/volume1/Docker/PhotoPrism/data/originals/
/volume1/Docker/PhotoPrism/data/storage/

Owner: User [1000]
Group: users [100]
My main image storage, where i have 2 folders.
1 for phone and 1 for Web (folder named Web)
/volume1/home/folder/MyPhoto/SmartUpload

**Reply by master3395** [reply]
I saw this in the docs.
It seems like it is not working, if i have it through the asustor app for now :/
https://docs.photoprism.app/getting-started/nas/asustor/

![image](https://github.com/user-attachments/assets/2f558adb-1335-412e-8228-2150b15b7a69)

**Comment by krakky4** [comment]
If running PhotoPrism in Docker, you should be able to pull the newest image from there and then just re-start the stack. No need for the Asustor App. _______von unterwegs gesendetAm 12.06.2024 um 00:03 schrieb Master3395 ***@***.***>:﻿

You're absolutely right! I'll get in touch with them to let them know there's a newer release.

Couldn't we also use the releases and manually install the app, if the APK was available on the github release also?

**Reply by master3395** [reply]
I know i can run it in a docker, but then I would have to re-create the docker, in order to update it, and so I would have to export all photos/videos before that time.
Takes too much stuff.
It's easier to just get the asustor app updated.
Because the storage isn't inside the app there.

**Reply by graciousgrey** [reply]
If the volume mounts are configured and all folders are mounted to persistent volumes you won't loose data when restarting the container.

https://docs.photoprism.app/getting-started/docker-compose/#volumes

**Comment by krakky4** [comment]
I am new to docker and don‘t know if I tinkered some kind of unusual setup. But when I re-create the docker with an updated image, my database remains the same. I was surprised myself about how easy this worked.Again, can’t vouch for anything here - just sharing my experience :)_______von unterwegs gesendetAm 17.06.2024 um 00:45 schrieb Master3395 ***@***.***>:﻿
I know i can run it in a docker, but then I would have to re-create the docker, in order to update it, and so I would have to export all photos/videos before that time.
Takes too much stuff.
It's easier to just get the asustor app updated.
Because the storage isn't inside the app there.
